src/JsonTxBuilderBase.sol

```diff
@@ -129,7 +129,7 @@ abstract contract JsonTxBuilderBase is CommonBase {
 
     /// @notice Returns a list of addresses which are expected to be in storage, but will not to have code on this
     ///         chain. Examples of such addresses include EOAs, predeploy addresses, and inbox addresses.
-    function getCodeExceptions() internal pure virtual returns (address[] memory) {
+    function getCodeExceptions() internal view virtual returns (address[] memory exceptions_) {
```

Comment:
I still get a warning for this function, not sure if there's a way to suppress it.